pyEcalVeto/mods/physTools.py

```diff
@@ -329,6 +332,56 @@
                                 441.6281859, 432.5241826, 455.2878243, 492.2888656, 502.6653722, 
                                 519.9101788, 539.1604349, 558.410691, 577.6609471, 596.9112032, 
                                 616.1614593, 635.4117154, 654.6619715, 673.9122276 ]
+'''
```

Comment:
we should make these a function in the future, so it's easier to change between geoms / energy. Right now having to uncomment and comment surely works but could be made nicer

Reply:
We can do this later too, doesnt have to happen in this PR